app/src/main/java/com/android/sample/model/image/ImageRepositoryFirestore.kt

```diff
@@ -140,4 +140,35 @@ constructor(private val firestore: FirebaseFirestore, private val storage: Fireb
         },
         onFailure)
   }
+
+  override fun removeAllActivityImages(
```

Comment:
The function is well-structured and easy to follow, with clear separation between listing images and deleting them.
Good job! However, you must add some logging in it so that it is more understandable, look at my proposals

Reply:
Thank you for your suggestions, very useful.